nipype/interfaces/mrtrix3/utils.py

```diff
@@ -224,13 +224,14 @@ class Generate5ttInputSpec(MRTrix3BaseInputSpec):
         "fsl",
         "gif",
         "freesurfer",
+        "hsvs",
         argstr="%s",
         position=-3,
         mandatory=True,
         desc="tissue segmentation algorithm",
     )
     in_file = File(
-        exists=True, argstr="%s", mandatory=True, position=-2, desc="input image"
+        exists=False, argstr="%s", mandatory=True, position=-2, desc="input image"
```

Comment:
So it seems like you need either a file or a directory, not a missing file.

```
in_file = traits.Either(
    File(exists=True),
    Directory(exists=True),
    argstr="%s", mandatory=True, position=-2, desc="input image / directory"
)

Reply:
Thanks! I added this solution now. :)